platform/ios/src/MGLMapView.h

```diff
@@ -358,6 +358,23 @@ MGL_EXPORT IB_DESIGNABLE
 - (void)setUserLocationVerticalAlignment:(MGLAnnotationVerticalAlignment)alignment animated:(BOOL)animated;
 
 /**
+ A Boolean value indicating whether the user location annotation may display a
+ permanent heading indicator.
+
+ Setting this property to `YES` causes the default user location annotation to
+ always show an arrow-shaped heading indicator, if heading is available. This
+ property does not rotate the map; for that, see
+ `MGLUserTrackingModeFollowWithHeading`.
+ 
+ This property has no effect when `userTrackingMode` is
+ `MGLUserTrackingModeFollowWithHeading` or
+ `MGLUserTrackingModeFollowWithCourse`.
+
+ The default value of this property is `NO`.
+ */
+@property (nonatomic, assign) BOOL showsUserHeadingIndicator;
```

Comment:
Should this property be inspectable? What happens if this property is set but `showsUserLocation` is not?

Reply:
> Should this property be inspectable?

Sure, that would be useful — I’ll do this.

> What happens if this property is set but `showsUserLocation` is not?

`showsUserHeadingIndicator` won’t do anything without `showsUserLocation=YES` — it doesn’t cause the user location annotation to appear. I considered having `showsUserHeadingIndicator` also implicitly enable `showsUserLocation` (the way our tracking modes do), but balked because it seemed a little too magical/indirect. Considering it again now, I could probably go either way. What do you think?

In any case, I’ll add documentation for this.

Reply:
Made this inspectable in 776a1c3 — renamed it for IB to try and disambiguate.

![screen shot 2017-09-01 at 3 11 08 pm](https://user-images.githubusercontent.com/1198851/29984342-04b950ba-8f28-11e7-895e-10240948846f.png)

Reply:
fa692f8: now with less truncation.

![screen shot 2017-09-01 at 3 24 11 pm](https://user-images.githubusercontent.com/1198851/29984933-c3d8a192-8f2a-11e7-9c1a-4364147da896.png)